web/router.ex

```diff
@@ -75,6 +75,7 @@ defmodule CoursePlanner.Router do
     end
 
     resources "/settings", SettingController, only: [:show, :edit, :update], singleton: true
+    resources "/aboutclientprogram", AboutClientProgramController, only: [:show], singleton: true
```

Comment:
i wanted to , but don't we wanna have an about page for our program ?
( i mean Dina's course planner ) . cause if no then I will, i was just not sure :P